File: collie/src/main/java/com/snail/collie/fps/ITrackFpsListener.java
package com.snail.collie.fps;

import android.app.Activity;

public interface ITrackFpsListener {

    void onFpsTrack(final Activity activity, final long currentCostMils, final long currentDropFrame, final boolean isInFrameDraw, long averageFps);

    void onANRAppear(final Activity activity, final long currentCostMils);
}

File: collie/src/main/java/com/snail/kotlin/fps/ITrackFpsListener.kt
package com.snail.kotlin.fps

import android.app.Activity

interface ITrackFpsListener {

    fun onFpsTrack(
        activity: Activity,
        currentCostMils: Long,
        currentDropFrame: Long,
        isInFrameDraw: Boolean,
        averageFps: Long
    )

    fun onANRAppear(activity: Activity?)

}